.github/workflows/release.yml

```
@@ -2,96 +2,38 @@ name: Release
 on:
   push:
     tags:
-      - '[2-9].[0-9]+.[0-9]+'
-      - '[2-9].[0-9]+.[0-9]+-*'
+      - '[0-9].[0-9]+.[0-9]+'
+      - '[0-9].[0-9]+.[0-9]+-*'
 env:
   DOTNET_NOLOGO: true
 jobs:
   release:
-    runs-on: windows-2019 # Code signing requirement https://github.com/NuGet/Home/issues/7939
+    runs-on: ubuntu-20.04 # Code signing requirement https://github.com/NuGet/Home/issues/7939
```

Windows was the requirement and no longer applies.
```suggestion
    runs-on: ubuntu-20.04
```